master/static/srv/get_model_versions.sql

```diff
@@ -36,3 +36,5 @@ SELECT
     version AS version,
     creation_time
     FROM c, m, mv
+    WHERE m.name = mv.model_name
```

Comment:
Hmm, isn't this condition always satisfied due to the WHERE clauses on `m` and `mv`?

Reply:
Yeah suppose it is, didn’t notice that

Reply:
I fixed this up by taking a step back and cleaning the whole query up a bit.. curiously I noticed a `modelv1.ModelVersion` should contain the `modelv1.Model` too but doesn't and never has.. I guess it's never used by the list endpoint, maybe just the get.. kinda funky design there. But the query should be overall more correct now. Could revisit the specifics of the protos later.